src/sentry/post_process_forwarder/post_process_forwarder.py

```diff
@@ -143,8 +142,10 @@ def create_with_partitions(
         commit: Commit,
         partitions: Mapping[Partition, int],
     ) -> ProcessingStrategy[KafkaPayload]:
+        from sentry.eventstream.kafka.dispatch import _get_task_kwargs_and_dispatch
```

Comment:
I don't think we should make this change.
1. Post process forwarder module should not depend on the eventstream. The dependency goes the other way - eventstream currently imports and calls PPF.
2. I think long term post process forwarding should be available on all Snuba topics where something needs to happen post ClickHouse insert - not just the ones that are the "eventstream" ones (which is only errors and transactions)

Reply:
gotcha. i made it so that the strategy factory can be reused for other datasets

the PostProcessForwarder class still has a bunch of event-specific logic in it, but that code should be deleted in 1-2 months anyway

I could've used `static_args` to pass the dispatch callback to the generic strategy, but the problem there is that I do not want to import random code into `sentry.consumers` either. So having dedicated classes per-consumer is more convenient, because those are imported lazily by string